src/olympia/reviewers/migrations/0013_backfill_autoapprovalsummary_score.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.17 on 2021-01-15 12:07
+
+from django.db import migrations
+from django.db.models import F, IntegerField, OuterRef, Subquery
+from django.db.models.functions import Cast
+
+from olympia.constants.base import STATUS_APPROVED
+from olympia.constants.scanners import MAD
+
+
+def forwards_func(apps, schema_editor):
+    AutoApprovalSummary = apps.get_model('reviewers', 'AutoApprovalSummary')
+    ScannerResult = apps.get_model('scanners', 'ScannerResult')
+
+    AutoApprovalSummary.objects.filter(
+        confirmed=None,
+        version__deleted=False,
+        version__addon__status=STATUS_APPROVED,
+        version__addon___current_version=F('version'),
+        version__scannerresults__score__gt=0,
+        version__scannerresults__scanner=MAD,
+    ).update(
+        # Django doesn't let us simply do F('version__scannerresults__score'),
+        # So we recompute that from a subquery. Note that we have to re-apply
+        # the filtering to make sure we get the right ScannerResult, and that
+        # we have to cast and multiply by 100 to match the way we store scores
+        # in AutoApprovalSummary.
+        score=Cast(
+            Subquery(
+                ScannerResult.objects.filter(
+                    version=OuterRef('version'),
+                    scanner=MAD,
+                ).values('score')[:1]
+            )
+            * 100,
```

Comment:
Should we do this here or rather in the view (template) layer? We're loosing some information here (maybe not important, though)